outputs/cloudevents/cloudevents.go

```diff
@@ -0,0 +1,67 @@
+package cloudevents
+
+import (
+	"context"
+	"errors"
+
+	ce "github.com/cloudevents/sdk-go/v2"
+	"github.com/cloudevents/sdk-go/v2/client"
+	"github.com/nsmith5/rekor-sidekick/outputs"
+)
+
+const (
+	driverName  = `cloudevents`
+	eventSource = `github.com/nsmith5/rekor-sidekick`
```

Comment:
I know I do this in the samples as is coded here, but in real production type systems, eventSource should also be appended with some identifying data to be able to find it. That could be as simple as the server url, or sometimes I make a guid and append it to the source path as you have.

Just something to think about, if you had two of these running and the client was getting events, how would it know which rekor-sidekick was invoking it?

Reply:
Ahhh I see. There is no unique information to making routing policy on...